README

```diff
@@ -313,8 +308,8 @@ Required for DreamPlug:
 
 Now, fetch the git source of freedom-maker and build the image you want:
 
-    $ git clone http://anonscm.debian.org/git/freedombox/freedom-maker.git freedom-maker
-    $ make -C freedom-maker dreamplug-image raspberry-image virtualbox-image
+    $ git clone https://github.com/freedombox/freedom-maker.git freedom-maker
+    $ make -C freedom-maker dreamplug raspberry virtualbox
```

Comment:
I think Alioth repository is still the primary repository.  GitHub pull requests were being accepted, however.  Both the repositories were up-to-date last I checked.  @petterreinholdtsen would know.

According to targets inside the Makefile, this command is correct:
```sh
$ make -C freedom-maker dreamplug-image raspberry-image virtualbox-image
```